package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -210,23 +253,150 @@ class WatsonCrickDist(NucPairDist):
+
+
+class MinorPairDist(NucPairDist):
+    r"""Minor-Pair basepair distance for selected residues over a trajectory.
+
+    Takes two lists of :class:`~MDAnalysis.core.groups.Residue` objects and
+    calculates the Watson-Crick distance between them over the trajectory.
+    Bases are matched by their index in the lists given as arguments.
+
+    Parameters
+    ----------
+    strand1: List[Residue]
+        First list of bases
+    strand2: List[Residue]
+        Second list of bases
+    oxygen2_name: str (optional)
+        Name of Oxygen 2 of nucleic acids
+        by default assigned to O2
+    carbon2_name: str (optional)
+        Name of Carbon 2 of nucleic acids
+        by default assigned to C2
+    guanine_name: str (optional)
+        Name of Guanine in topology
+        by default assigned to G
+    adenine_name: str (optional)
+        Name of Adenine in topology
+        by default assigned to G
+    uracil_name: str (optional)
+        Name of Uracil in topology
+        by default assigned to U
+    tymine_name: str (optional)
+        Name of Thymine in topology
+        by default assigned to T
+    cytosine_name: str (optional)
+        Name of Cytosine in topology
+        by default assigned to C
+    **kwargs: dict
+        arguments for :class:`~MDAnalysis.analysis.base.AnalysisBase`
+
+    Attributes
+    ----------
+    results: numpy.ndarray
+        first index is selection second index is time
+    results.times: numpy.ndarray
+        times used in analysis
 
-            sel1.append(s[0].atoms.select_atoms(f'name {a1}'))
-            sel2.append(s[1].atoms.select_atoms(f'name {a2}'))
+    Raises
+    ------
+    ValueError
+        if the residues given are not amino acids
+    ValueError
+        if the selections given are not the same length
+
+    """
+
+    def __init__(self, strand1: List[Residue], strand2: List[Residue],
+                 oxygen2_name: str = 'O2', carbon2_name: str = "C2",
+                 guanine_name: str = 'G', adenine_name: str = 'A', uracil_name: str = 'U',
+                 tymine_name: str = 'T', cytosine_name: str = 'C',
+                 **kwargs) -> None:
+        selections: Tuple[List[mda.AtomGroup], List[mda.AtomGroup]] = self.select_strand_atoms(
+            strand1, strand2, carbon2_name, oxygen2_name,
+            guanine_name=guanine_name, adenine_name=adenine_name,
+            tymine_name=tymine_name, uracil_name=uracil_name, cytosine_name=cytosine_name
+        )
+
+        super(MinorPairDist, self).__init__(selections[0], selections[1], **kwargs)
+
+
+
+class MajorPairDist(NucPairDist):
+    r"""Minor-Pair basepair distance for selected residues over a trajectory.
+
+    Takes two lists of :class:`~MDAnalysis.core.groups.Residue` objects and
+    calculates the Watson-Crick distance between them over the trajectory.
+    Bases are matched by their index in the lists given as arguments.
```

Comment:
How is `MajorPairDist` different from `WatsonCrickDist`? The docs do not say. Please add details to the docs to describe the difference.

Reply:
Start by copying the original docs eg https://docs.mdanalysis.org/stable/documentation_pages/analysis/nuclinfo.html#MDAnalysis.analysis.nuclinfo.major_pair

Also, the docs here incorrectly say "Minor-Pair".

Reply:
Explicitly state what distances are calculated.